docs/Using-EthSigner/Using-EthSigner.md

```diff
@@ -15,6 +16,55 @@ by EthSigner.
 !!! note 
     EthSigner supports the JSON-RPC service over HTTP only. 
 
+# eea_sendTransaction 
+
+Creates and signs a [private transaction](https://docs.pantheon.pegasys.tech/en/stable/Privacy/Privacy-Overview/)
+using the [keystore account](../Using-EthSigner/Getting-Started.md#create-password-and-key-files). 
+
+EthSigner submits the signed transaction to Pantheon using [`eea_sendRawTransaction`](https://docs.pantheon.pegasys.tech/en/stable/Reference/Pantheon-API-Methods/#eea_sendrawtransaction). 
+
+**Parameters**
+
+Transaction object for private transactions: 
+
+| Key             | Type                | Required/Optional                  | Value                                                                                                                         |
+|-----------------|--:-:----------------|----------------------------------- |-------------------------------------------------------------------------------------------------------------------------------|
+| **from**        | Data, 20&nbsp;bytes | Required                           | Address of the sender. Must be the address of the keystore account.                                                                                                        |
+| **to**          | Data, 20&nbsp;bytes | Not required for contract creation | `null` for contract creation transaction. Contract address for contract invocation transactions.                                                           |
+| **gas**         | Quantity            | Optional                           | Gas provided by the sender. Default is `90000`.                                                                               |
+| **gasPrice**    | Quantity            | Optional                           | Gas price provided by the sender in Wei. Default is `0`.                                                                      |
+| **nonce**       | Quantity            | Optional                           | Number of transactions sent from the `from` account before this one.                                                                    |
+| **data**        | Quantity            | Optional                           | Compiled contract code or hash of the invoked method signature and encoded parameters.                                        |
+| **privateFrom** | Data, 20&nbsp;bytes | Required                           | Orion address of the sender                                                                                                         |
+| **privateFor**  | Array of data       | Required                           | Orion addresses of recipients                                                                                                       |
+| **restriction** | String              | Required                           | Must be [`restricted`](https://docs.pantheon.pegasys.tech/en/stable/Privacy/Privacy-Overview/#private-transaction-attributes) |
```

Comment:
Does it have to be "restricted"? I thought we could use "unrestricted" too?

Reply:
Has to be restricted - Pantheon doesn't support unrestricted private transactions (explained in the overview)